week2/assignments/01_sign_up_styling.html

```diff
@@ -7,55 +7,46 @@
 
     <style>
       * {
-        /*
-          - May be you want to add "border-box" for "box-sizing" property so that
-            padding does not affect width of HTML elements
-          - Also, better to Reset margin and padding
-       */
+        box-sizing: border-box;
+        margin: 0 auto;
       }
+
       body {
-        /* Probably want to set some font-family as default Times New Roman looks oldish */
-        /* Set background color to #ff7070 */
+       font-family: arial;
```

Comment:
@angkamilama  this should be `Arial, Helvetica, sans-serif`. 
This is the order of font that will be used by the browser. If Arial is not there then it will use Helvetica else sans-serif or system font.